task11/queries/query_5.sql

```diff
@@ -0,0 +1,5 @@
+SELECT NAME
+FROM user
+WHERE (SELECT COUNT(post.POST_ID) 
```

Comment:
Can be + 
But also grouping + joining features can solve this task (in the real world in some cases group aggregation based operations works faster)